src/elgamal.rs

```diff
@@ -69,6 +70,52 @@ pub struct ElgamalCipher {
     delta: JubJubExtended,
 }
 
+impl Serializable<64> for ElgamalCipher {
+    type Error = BytesError;
+
+    /// Serialize the cipher into bytes
+    fn to_bytes(&self) -> [u8; Self::SIZE] {
+        let gamma: JubJubAffine = self.gamma.into();
+        let gamma = gamma.to_bytes();
+
+        let delta: JubJubAffine = self.delta.into();
+        let delta = delta.to_bytes();
+
+        let mut bytes = [0u8; Self::SIZE];
+
+        bytes[..32].copy_from_slice(&gamma);
+        bytes[32..].copy_from_slice(&delta);
+
+        bytes
+    }
+
+    /// Deserialize from a [`ElgamalCipher::to_bytes`] construction
+    fn from_bytes(bytes: &[u8; Self::SIZE]) -> Result<Self, Self::Error> {
+        let mut gamma = [0u8; 32];
+        let mut delta = [0u8; 32];
+
+        gamma.copy_from_slice(&bytes[..32]);
+        delta.copy_from_slice(&bytes[32..]);
+
+        let gamma = JubJubAffine::from_bytes(&gamma);
```

Comment:
Maybe do a `JubJubAffine::from_slice(&bytes[..32])` instead of executing a new copy?